### PR TITLE
fix: set Microsoft log level to warning

### LINKED ISSUE #354: Reduce verbosity for telemetry by Microsoft
**Is your feature request related to a problem? Please describe.**
Telemetry should be actionable and not generate noise, by default Microsoft telemetry is added which is considered to be noise.

**Describe the solution you'd like**
Reduce verbosity for telemetry by Microsoft by reducing it to Warning-level only.

https://github.com/arcus-azure/arcus.templates/blob/master/src/Arcus.Templates.WebApi/Startup.cs#L314

**Describe alternatives you've considered**
N/A

**Additional context**
This generates a lot of logging statements when executing an HTTP endpoint.

### PR DESCRIPTION
**Pull request by stijnmoreels**
To reduce logs from Microsoft, we should set it to warning.

Closes #354

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]

**Review by fgheysels — APPROVED** [review]
Thanks !

File: src/Arcus.Templates.WebApi/Startup.cs [hunk 1]
```diff
@@ -311,7 +311,7 @@ private LoggerConfiguration CreateLoggerConfiguration(IServiceProvider servicePr
             
             return new LoggerConfiguration()
                 .MinimumLevel.Debug()
```
Comment (fgheysels) [inline src/Arcus.Templates.WebApi/Startup.cs:313]: Is it a good idea to set the loglevel to Debug ?  I mean, Arcus is a library, customers who use this library in their project might get overwhelmed with Arcus logs that they do not want.

Comment (stijnmoreels) [reply src/Arcus.Templates.WebApi/Startup.cs:313]: Or set it on Debug when the configuration is Debug 😄 .

Comment (tomkerkhove) [reply src/Arcus.Templates.WebApi/Startup.cs:313]: +1. I'd use info by default

Comment (fgheysels) [reply src/Arcus.Templates.WebApi/Startup.cs:314]: I'd set it to Info, regardless if the configuration is debug or release.

**Pull request merged**